server/commands/joinGame.js

```diff
@@ -0,0 +1,28 @@
+const joinGame = (io, socket, games) => socket.on('joinGame', function(data) {
+    const game = games[data.gameId];
+    console.log('join', game, data);
+    if (!game || game.guest) {
```

Comment:
That only exists 1 guest.
The second time that enters in the same url we return an error